tribe-ext-attendees-table-dups/index.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+* Plugin Name: Event Tickets Plus Extension: Show/Hide Duplicates in Attendees Table
+* Description: Makes it easy to identify tickets from duplicate order numbers (e.g. 4 WooCommerce tickets purchased in a single transaction) at each event's Attendees Table view. Displays the count of total tickets per order and adds a button to show/hide duplicates. Note: the "Check in" button still applies per-ticket and not per-order.
+* Version: 1.0
+* Extension Class: Tribe__Extension__Attendees_Table_Duplicates
+* Author: Modern Tribe, Inc.
+* Author URI: http://m.tri.be/1971
+* License: GPLv2 or later
+* License URI: https://www.gnu.org/licenses/gpl-2.0.html
+*/
+
+// 2017-02-08 Cliff created for https://theeventscalendar.com/support/forums/topic/keeping-orders-together-on-attendee-list/
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+/**
+* Extension main class, class begins loading on init() function.
+*/
+class Tribe__Extension__Attendees_Table_Duplicates extends Tribe__Extension {
+	
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 */
+	public function construct() {
+		// Each plugin required by this extension
+		$this->add_required_plugin( 'Tribe__Tickets_Plus__Main', '4.3' ); // does not work with RSVP tickets and was really only tested with WooCommerce tickets
+		
+		// Set the extension's TEC URL
+		$this->set_url( 'https://theeventscalendar.com/extensions/attendees-table-show-hide-duplicates/' );		
+	}
+	
+	/**
+	 * Extension initialization and hooks.
+	 */
+	public function init() {
+		add_action( 'tribe_events_page_tickets-attendees', array( $this, 'output' ) );
+	}
+	
+	/**
+	 * Logic for the output
+	 *
+	 * @return string
```

Comment:
As you brought out in your DM, this can be removed since it doesn't return anything.